debian/base/Dockerfile.rpm

```diff
@@ -72,22 +78,35 @@ RUN echo "Installing Zulu OpenJDK ${ZULU_OPENJDK_VERSION}" \
     && yum -q -y update \
     && yum -q -y install zulu-${ZULU_OPENJDK_VERSION}
 
-RUN echo "===> Adding confluent repository...${CONFLUENT_RPM_REPO}/rpm/${CONFLUENT_MAJOR_VERSION}.${CONFLUENT_MINOR_VERSION}" \
-    && rpm --import ${CONFLUENT_RPM_REPO}/rpm/${CONFLUENT_MAJOR_VERSION}.${CONFLUENT_MINOR_VERSION}/archive.key
-
-RUN printf "[Confluent.dist] \n\
+RUN echo "===> Adding confluent repository...${CONFLUENT_PACKAGES_REPO}/rpm/${CONFLUENT_MAJOR_VERSION}.${CONFLUENT_MINOR_VERSION}" && \
```

Comment:
You might want to put all this in a bash file, and then just have RUN call the bash file to avoid having to do all the escapes and also improve readability. Also ok if you want to leave it for now.